### PR TITLE
API 37106 update problematic regex in v2 only

### PR DESCRIPTION
**Pull request by rockwellwindsor-va**
## Summary
* Updates on REGEX addresses where more then 30 characters are accepted <- IMPORTANT as you wil see the old validation mentioned in the ticket, make sure to verify the max limit accepted.
* Compiles swagger docs to reflect change

## Related issue(s)
[API-37106](https://jira.devops.va.gov/browse/API-37106)

## Testing done
* The tests should remain and behave the same with this update

## What areas of the site does it impact?
	modified:   modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json
	modified:   modules/claims_api/app/swagger/claims_api/v2/production/swagger.json
	modified:   modules/claims_api/config/schemas/v2/526.json
	modified:   modules/claims_api/config/schemas/v2/generate_pdf_526.json

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

* If anyone can find a value for address that would be accepted by the previous REGEX but not the new one that would be awesome.  I have not been able to come up with anything.

### CODE REVIEW

File: modules/claims_api/app/swagger/claims_api/v2/production/swagger.json [hunk 1]
```diff
@@ -1405,7 +1405,7 @@
                                     "city": {
                                       "description": "City for the Veteran's current mailing address.",
                                       "type": "string",
-                                      "pattern": "^([-a-zA-Z0-9'.#]([-a-zA-Z0-9'.# ])?)+$",
+                                      "pattern": "^$|^([-a-zA-Z'.' ])+$",
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:1408]: seems like we have a few patterns for the city matching.... is there a reason for the slight discrepancy?
```
^$|^([-a-zA-Z'.' ])+$
^$|([-a-zA-Z'.' ])+$
```

Comment (rockwellwindsor-va) [reply modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:1408]: No I think I just deleted to far in, updating that now

**Review by FonzMP — APPROVED** [review]
:shipit:

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**